documentation/whatsnew/server/server-news-4.3.0.Alpha1.adoc

```diff
@@ -0,0 +1,30 @@
+= Server What's New in 4.3.0.Alpha1
+:page-layout: whatsnew
+:page-component_id: server
+:page-component_version: 4.3.0.Alpha1
+:page-feature_jbt_only: true
+:page-product_id: jbt_core 
+:page-product_version: 4.3.0.Alpha1
+
+
+== IP6 Support Added
+
+For JBoss 7 and Wildfly installations, using hostnames with an IPv6 format often failed to connect, show webpages, or even verify the server's state. Several changes have made this work as expected. With updated wildfly jars, management commands will execute without issue. In addition, our tools will now automatically recognize host names in an IPv6 pattern, and update your launch configuration's various launch arguments to ensure that the server starts properly and with all expected flags set. 
+
+image::images/ip6_host.png[Setting the IPv6 Host]
+image::images/ip6_launch_config.png[Launch Configuration for IPv6 Hosts]
+
+
+related_jira::JBIDE-19113[] - Update of wildfly jars
+related_jira::JBIDE-12352[] - Remote Management Connections to Wildfly IPv6
+related_jira::JBIDE-12351[] - JMX connections to Wildfly IPv6
+related_jira::JBIDE-12282[] - Wildfly server fails to Stop IPv6
+
+== Project Archives, ant usecase deprecated
+
+In previous releases, Project Archives was designed in such a way that the core plugin would be compiled into multiple jars, one of which had minimal dependencies on Eclipse, allowing it to be used from ant. This usecase was fairly complicated to get working for the user, and involved locating and adding several eclipse jars to the ant classpath, making use of a custom ant task, and adding a significant amount of configuration in your build.xml file to properly map details like a project's location. 
+
+Despite attempting to be portable, persistable, and shareable, this usecase never really lived up to any of those goals. While the eclipse tooling could simply use a project name in its internal metadata, allowing the user to keep those projects wherever they want while using eclipse to resolve their location, the ant usecase required hard-coded location mappings for each project in the build.xml, limiting its usefulness in a shared environment. 
```

Comment:
Had nothing to do with linked folders, though to be honest we didn't even have ANY way to handle that. No. What i mean here is that the build.xml would need to map the project name to its location. The project archives file .packages worked by having first segment of a path be the project name.    In eclipse we can just do workspace.getProject(name).getLocation();   In ant, there's no way to do that, since a project is an eclipse construct. The user's build.xml would need to set properties that mapped a project name to its location. This meant the build.xml could not be effectively shared, since every user may choose to check out the project at a different fs path.